media/com_templates/js/admin-template-toggle-switch.es6.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+function showDiffChanged() {
```

Comment:
This needs to be wrapped in it's own function like in the other ES6 js files.

Reply:
@laoneo Mean `=>` arrow function?